### PR TITLE
feat(docker): add docker file to run it inside a container

### PR DESCRIPTION
**Pull request by davinkevin**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+COPY requirements.txt /fitbit-googlefit/requirements.txt
+RUN pip install -r requirements.txt
+
+COPY . /fitbit-googlefit/
+
+CMD python3 app.py
+
+# Use with the following command: 
+# $ docker run --rm -it -v /path/to/auth/fitbit.json:/fitbit-googlefit/auth/fitbit.json -v /path/to/auth/google.json:/fitbit-googlefit/auth/google.json davinkevin/fitbit-googlefit
```
Comment (praveendath92) [inline Dockerfile]: Could you fix `davinkevin/fitbit-googlefit` in this comment?

Comment (davinkevin) [reply Dockerfile]: Will you deploy it on the Dockerhub ? If yes, I can put your repo name `praveendath92` as official image.

**Comment by davinkevin** [comment]
Of course you can, if you launch the container with the following command, you will have a bash ready in the project folder. So, it's like if you are at the step 2 of the setup.

But, of course, you have to run the command in `--console` mode.

**Comment by davinkevin** [comment]
Any news about this PR ?

**Comment by davinkevin** [comment]
Still no news, I will close it if not useful.

**Comment by davinkevin** [comment]
Up ?

**Comment by praveendath92** [comment]
@davinkevin Sorry for such a long delay. I'm going ahead and merging this contribution. 

Wondering if it would helpful to add some instructions on how to do authentication using the same container - in the spirit of not having to setup python & dependencies on the host just for authentication.

**Pull request merged**